File: src/main/java/com/lawencon/elearn/dto/examcollection/ExamCollectionUpdateResDto.java
package com.lawencon.elearn.dto.examcollection;

public class ExamCollectionUpdateResDto {
	private ExamCollectionUpdateDataResDto data;
	private String message;
	public ExamCollectionUpdateDataResDto getData() {
		return data;
	}
	public void setData(ExamCollectionUpdateDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}

File: frontend/src/app/dto/exam-collection/exam-collection.update.res.dto.ts
import { ExamCollectionUpdateDataResDto } from "./exam-collection.update.data.res.dto";

export class ExamCollectionUpdateResDto{
    private _data! : ExamCollectionUpdateDataResDto
	private _message! : string

    public get data(): ExamCollectionUpdateDataResDto {
        return this._data;
    }

    public set data(data: ExamCollectionUpdateDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }
    
}